codegen/arrai/grpc_client.arrai

```diff
@@ -15,6 +15,17 @@ let go = //{./go};
     $`
         ${go.prelude(app, (clientDeps => $`${basepath}/${.import}`) | go.pbPackage(app))}
 
+        // Service interface for ${appname}.
+        type Service interface {
```

Comment:
Can you include the generated code that changes as a result of this change.

Reply:
I think the generated code is in the .gitignore file and not added to the repo.